boards/cats_rev1Pro/src/util/enum_str_maps.c

```diff
@@ -0,0 +1,51 @@
+/*
+ * CATS Flight Software
+ * Copyright (C) 2022 Control and Telemetry Systems
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+#include "config/cats_config.h"
+#include "config/globals.h"
+
+#include <stdio.h>
+#include <stdlib.h>
+
+const char* fsm_map[14] = {"INVALID",  "MOVING",      "READY",      "THRUSTING_1", "THRUSTING_2",
+                           "COASTING", "TRANSONIC_1", "SUPERSONIC", "TRANSONIC_2", "APOGEE",
+                           "DROGUE",   "MAIN",        "TOUCHDOWN",  "HEHE"};
+
+const char* const boot_state_map[6] = {
+    "CATS_INVALID", "CATS_IDLE", "CATS_CONFIG", "CATS_TIMER", "CATS_DROP", "CATS_FLIGHT",
+};
+
+const char* const event_map[9] = {
+    "MOVING", "READY", "LIFTOFF", "MAX_V", "APOGEE", "POST_APOGEE", "TOUCHDOWN", "CUSTOM_1", "CUSTOM_2",
+};
+
+const char* const action_map[17] = {
+    "NONE",   "DELAY",   "HC_ONE",  "HC_TWO",    "HC_THREE",  "HC_FOUR",     "HC_FIVE",    "HC_SIX",   "LL_ONE",
+    "LL_TWO", "LL_TREE", "LL_FOUR", "SERVO_ONE", "SERVO_TWO", "SERVO_THREE", "SERVO_FOUR", "RECORDER",
+};
+
+char* recorder_speed_map[NUM_REC_SPEEDS] = {};
+
+void init_recorder_speed_map() {
```

Comment:
Should this really be here?

Reply:
Yeah, I think so, since it's initializing this enum -> str map. The other code doesn't have to know how it is being done.